src/tests/functional/shared_test_classes/src/single_op/dft.cpp

```diff
@@ -3,11 +3,61 @@
 //
 
 #include "shared_test_classes/single_op/dft.hpp"
-
 #include "common_test_utils/node_builders/dft.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "shared_test_classes/base/utils/ranges.hpp"
 
 namespace ov {
 namespace test {
+static inline void set_real_number_generation_data(utils::InputGenerateData& inGenData) {
+    inGenData.range = 8;
+    inGenData.resolution = 32;
+}
+
+void DFTLayerTest::generate_inputs(const std::vector<ov::Shape>& targetInputStaticShapes) {
```

Comment:
why it is required?

Reply:
It's needed for fp16 precision. While 125993 does not implemented.